guides/release-notes.md

```diff
@@ -7,6 +7,25 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New API: Dashboard Elements
+
+`Wix Editor Elements ($w)`
+
+The new [Dashboard Elements](https://www.wix.com/velo/reference/$w/dashboardbutton) API allows you to create customized, interactive dashboards for your users.
```

Comment:
> Dashboard Elements API

This isn't a single 'dashboard elements' API, but rather a set of APIs for dashboard elements.